app/controllers/invite_controller.rb

```diff
@@ -0,0 +1,22 @@
+class InviteController < ApplicationController
+  def new
+  end
+
+  def create
+    @github_facade = GithubFacade.new(current_user)
+    invitee = @github_facade.invitee(invite_params[:github_handle])
+    if invitee.email && invitee.name
+      UserMailer.invite(invitee.email, invitee.name, current_user).deliver_later
```

Comment:
We might want to refactor how we pass information to the mailer views in the future